images/linux/Ubuntu2004-README.md

```diff
@@ -257,14 +258,14 @@ apt-get update
 - Android SDK Build-Tools 27.0.1
 - Android SDK Build-Tools 27.0.0
 - Android NDK 21.3.6528147
-- Az Module (4.4.0)
+- Az Module (4.5.0)
 - Cached container images
+  - node:10 (Digest: sha256:cf3ee6a5a1b1916c7a2e4fb51eb7ecba1afe186739677d62e9c1bb2cb1c7d6b0)
+  - node:12 (Digest: sha256:d0738468dfc7cedb7d260369e0546fd7ee8731cfd67136f6023d070ad9679090)
+  - buildpack-deps:stretch (Digest: sha256:34a18637ed801407f7a17a29575e82264fb0818f9b6a0c890f8a6530afea43dc)
+  - debian:9 (Digest: sha256:335ecf9e8d9b2206c2e9e7f8b09547faa9f868e694f7c5be14c38be15ea8a7cf)
+  - debian:8 (Digest: sha256:8a0f2603166345b4d7bbf4842137b2ffcb492ece20d15f963f08aa26670f82c7)
```

Comment:
Why is this info always in the different places of the section? 
It creates additional noise in diff. Should we fix it as a low priority?